### PR TITLE
Add the profile of valentin

### PR DESCRIPTION
**Pull request by neoflex**

### CODE REVIEW

File: _attendees/valentin.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+name: Valentin Grouès
+position: Sitting
```
Comment (laurentheirendt) [inline _attendees/valentin.md]: 🏃 or 💺 ? That's the question.

File: _attendees/valentin.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+institution: University of Luxembourg
+ghurl: https://github.com/neoflex
+ghimg: https://avatars2.githubusercontent.com/u/888151
+---
+
+<a href="{{ page.ghurl }}"><img src="{{ page.ghimg }}" height="150px"/></a>
+
+# {{ page.name }}
+
+{{ page.name }} is a {{ page.position | downcase }} in the {{ page.group }} at the {{ page.institution }}.
```
Comment (laurentheirendt) [inline _attendees/valentin.md]: You could write:
```
{{ page.name }} is {{ page.position | downcase }} in the {{ page.group }} at the {{ page.institution }}.
```

**Review by laurentheirendt — CHANGES_REQUESTED** [review]

**Review by laurentheirendt — APPROVED** [review]

File: _attendees/valentin.md (hunk 2 above)
Comment (neoflex) [reply _attendees/valentin.md]: done

**Comment by laurentheirendt** [comment]
Very well done, @neoflex 👍

**Pull request merged**